src/content/en/updates/2019/06/image-support-for-async-clipboard.md

```diff
@@ -0,0 +1,299 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: The Async Clipboard API, the text-focused portion of which we’ve shipped in Chrome 66, has learned a new trick. As of Chrome 76, it now also handles images.
+
+{# wf_published_on: 2019-06-30 #}
+{# wf_updated_on: 2019-06-30 #}
+{# wf_featured_image: /web/updates/images/generic/photo.png #}
+{# wf_tags: chrome76,cutandcopy,execcommand,input,clipboard #}
+{# wf_featured_snippet: The Async Clipboard API, the text-focused portion of which we’ve shipped in Chrome 66, has learned a new trick. As of Chrome 76, it now also handles images. #}
+{# wf_blink_components: Platform>Apps>AppLauncher>Install #}
+
+# Image Support for the Async Clipboard API {: .page-title }
+
+{% include "web/_shared/contributors/thomassteiner.html" %}
+
+In Chrome&nbsp;66, we shipped the [text-focused portion](/web/updates/2018/03/clipboardapi)
+of the Asynchronous [Clipboard API](https://w3c.github.io/clipboard-apis/)
+that replaced the previous method that was based on
+[`Document.execCommand()`](https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand)
+to [cut and copy](/web/updates/2015/04/cut-and-copy-commands) contents on the web.
+Now in Chrome&nbsp;76, we are shipping the *image-focused* portion of the Asynchronous Clipboard
+API.
+
+Before I dive into what is shipping now, let me briefly look back at how the Asynchronous Clipboard
+API works.
```

Comment:
This is the only 1st-person sentence I see in this article. Not an issue necessarily, but just wondering if it was intentional.

Reply:
Oops, just finished a long 1st person voice article (https://web.dev/prefers-color-scheme/).  Fixed here now with https://github.com/google/WebFundamentals/pull/7823/commits/eb85ec37f3cf5e3309cf3723abd269225e80cefa.